include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -127,6 +127,13 @@ inline const OrtApi& GetApi() noexcept { return *Global<void>::api_; }
 /// <returns>version string major.minor.rev</returns>
 std::string GetVersionString();
 
+/// <summary>
+/// This function returns the onnxruntime build information: including git branch,
+/// git commit id, build type(Debug/Release/RelWithDebInfo) and cmake cpp flags.
+/// </summary>
+/// <returns>string</returns>
+std::string GetBuildInfoString();
```

Comment:
It would be helpful to make it available in the python bindings as well. Any test to check its value?

Reply:
That's a good point. I will take a look at how to add the python binding and the test to cover.

Reply:
@xadupre I have added the build info to python bindings. The figure below shows the effect by some temporary code.
![image](https://user-images.githubusercontent.com/19584326/234500734-05d19f28-33c9-403d-9852-70bb107ea8bd.png)